dbt_meshify/storage/dbt_project_editors.py

```diff
@@ -71,48 +152,39 @@ def copy_packages_dir(self) -> None:
         """
         raise NotImplementedError("copy_packages_dir not implemented yet")
 
-    def update_dependencies_yml(self) -> None:
-        try:
-            contents = self.file_manager.read_file(Path("dependencies.yml"))
-        except FileNotFoundError:
-            contents = {"projects": []}
-
-        contents["projects"].append({"name": self.subproject.name})  # type: ignore
-        self.file_manager.write_file(Path("dependencies.yml"), contents, writeback=True)
-
     def update_child_refs(self, resource: ManifestNode) -> None:
         downstream_models = [
             node.unique_id
-            for node in self.subproject.manifest.nodes.values()
+            for node in self.project.manifest.nodes.values()
             if node.resource_type == "model" and resource.unique_id in node.depends_on.nodes  # type: ignore
         ]
         for model in downstream_models:
-            model_node = self.subproject.get_manifest_node(model)
+            model_node = self.project.get_manifest_node(model)
             if not model_node:
                 raise KeyError(f"Resource {model} not found in manifest")
             meshify_constructor = DbtMeshConstructor(
-                project_path=self.subproject.parent_project.path, node=model_node, catalog=None
+                project_path=self.project.parent_project.path, node=model_node, catalog=None  # type: ignore
```

Comment:
What's going on with the typing for `DbtMeshConstructors`?

Reply:
i went pretty wild with type ignoring 😃 

this one looks like a miss, but was having trouble convinving mypy that the `DbtSubprojectCreator` could only get a `DbtSubproject`

Reply:
i thought the code on 102 would be sufficient to suppress the type hints, but alas!